src/java.base/windows/classes/sun/nio/fs/WindowsNativeDispatcher.java

```diff
@@ -1094,9 +1094,7 @@ static NativeBuffer asNativeBuffer(String s) throws WindowsException {
     private static native void initIDs();
 
     static {
-        // nio.dll has dependency on net.dll
         jdk.internal.loader.BootLoader.loadLibrary("net");
```

Comment:
...do we still need net here?

Reply:
We don't need it in `libjava`, but `NTLMAuthentication` was (perhaps unwittingly) relying on it to load `net.dll` during boot phase 2. I'll move the load to a more appropriate location in the next commit.

Reply:
Moved to `NTLMAuthentication` static initializer. 853d349

Reply:
@dfuch Would you please check whether the change at line 71 in the updated version of `NTLMAuthentication` is the correct place for this load?

Reply:
hmm...  I don't see any issue in adding the load call to `NTLMAuthentication` but I'm surprised that it's even needed: I'd expect that libnet would have been loaded before we reach NTLMAuthentication.

Reply:
I wonder - do you see any failure if you don't load libnet from there?

Reply:
Yes, there was an `UnsatisfiedLinkError` in the native method `isTrustedSiteAvailable()` in `NTLMAuthentication`.

Reply:
> [...] I'd expect that libnet would have been loaded before we reach NTLMAuthentication.

In the (as of now) penultimate webrev 4f47d5a (Merge), `WindowsNativeDispatcher` loaded it during boot phase 2. I think that without these changes it is likely loaded by `IOUtil`.

Reply:
Do we know what code loaded NTLMAuthentication? I'd expect it to be loaded by HttpURLConnection, which should have triggered the loading of libnet long before it cares about NTLM.

Reply:
I would have to check. The failure I observed occurred in both of these tests
```
test/jdk/sun/net/www/protocol/http/NoNTLM.java
test/jdk/sun/net/www/protocol/http/TestTransparentNTLM.java
```
and nowhere else.

Reply:
I see. The test does:

            `Class<?> ntlmProxyClass = Class.forName("sun.net.www.protocol.http.NTLMAuthenticationProxy", true, NoNTLM.class.getClassLoader());
`
so that explains it.

In this case I believe it's fair enough to have NTLMAuthentication trigger the loading of libnet if not loaded already -since we need that library to perform the class initialization properly.

What you have is good to me.

Reply:
Good! Thanks for the assistance.